### DISCUSSION TITLE
6.2 拜占庭将军问题

### DISCUSSION THREAD

**Discussion by beiciye-h-20230331** [isno/theByteBook#123] [General]
https://www.thebyte.com.cn/consensus/The-Byzantine-General-Problem.html#_2-%E5%8F%A3%E4%BF%A1%E6%B6%88%E6%81%AF%E5%9E%8B%E8%A7%A3%E5%86%B3%E6%96%B9%E6%A1%88

当增加1个叛将时，需增加3个以以上忠将。    倘若，有1叛将，4个忠将。   叛将为指挥官，给2个传递进攻，2个传递撤退。  这种场景下会怎么处理呢

**Comment by isno** [comment]
忠将之间无法形成共识，这种情况在拜占庭将军问题属于无效操作。但在 Paxos 和 raft 中投票流程中，这种场景叫 split votes，你可以继续看 raft 中的篇章，有一点点这方面的说明。
